services/QuillLMS/app/controllers/grades_controller.rb

```diff
@@ -46,7 +46,7 @@ def tooltip
       percentage: activity_session.percentage,
       description: activity_session.activity.description,
       due_date: unit_activity.due_date,
-      completed_at: activity_session.completed_at.to_i + current_user.utc_offset.seconds,
+      completed_at: Time.at(activity_session.completed_at.to_i + current_user.utc_offset.seconds),
```

Comment:
Quick note, I think that because of the nature of Datetime math in Ruby, you can also just remove `to_i` so that you add seconds directly to the `completed_at` value.  But this works, too.

Reply:
this is cleaner, thank you.